tests/test_phreeqc.py

```diff
@@ -257,3 +257,29 @@ def test_equilibrate(s1, s2, s5_pH, s6_Ca, caplog):
     s6_Ca.equilibrate()
     assert s6_Ca.get_total_amount("Ca", "mol").magnitude != initial_Ca
     assert np.isclose(s6_Ca.charge_balance, 0, atol=1e-8)
+
```

Comment:
Tests look good to me. Can you please add a 1 line comment above each of the `assert` statements (in tests 2-4) explaining how you arrived at the correct test value?

Something like "p_N2 = 0.78, Henry's law = XXX, dissolved N2 = YYYY"